src/main/java/model/FruitRecord.java

```diff
@@ -0,0 +1,68 @@
+package model;
+
+public class FruitRecord {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitRecord(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+
+    public enum Operation {
```

Comment:
Consider moving the `Operation` enum to a separate file. This will make the code more organized and adhere to the single responsibility principle, as the enum can be used by other classes as well.